cmd/gen/makefile/command.go

```diff
@@ -17,7 +17,9 @@ There are different generation flavours:
 
   - app - project containing a helm chart
   - cli - project released with a downloadable binary
-  - generic - everything else, i.e a project which simply needs to be released`
+  - generic - everything else, i.e a project which simply needs to be released
+  - k8sapi - project containing a Kubernetes API
```

Comment:
what would projects like `release-operator` choose? I mean a project that contais both go code with business logic, and a Kubernetes API.

Reply:
A repository can have multiple flavors separated by commas. See https://github.com/giantswarm/github/blob/master/repositories/team-honeybadger.yaml#L121 for example.

Reply:
I did not know that, cool! Thanks